CONTRIBUTING

```diff
@@ -0,0 +1,89 @@
+# Contributing to Dipy
```

Comment:
Thanx for starting to write such a document. Shouldn't this be an rst file in the dipy/doc folder? 
Here you wrote a markdown file but we don't have any markdown documents in dipy. I have a feeling this is in the wrong format and possibly folder. This information should  be rendered by sphinx or something in the docs. You can also have a copy in the setup.py folder but that is secondary.